test/TestHelpers.cs

```diff
@@ -81,7 +83,8 @@ internal static class TestHelpers
             string instanceId,
             bool extendedSessions,
             string[] orchestratorFunctionNames,
-            string activityFunctionName = null)
+            string activityFunctionName = null,
+            bool logReplayEvents = true)
```

Comment:
I think I mentioned this in another iteration, but I don't think it's necessary to create this new `logReplayEvents` parameter. The existing `extendedSessions` parameter basically doesn't the same thing already. Can we just rename that parameter to something like `filterOutReplayLogs`?

Reply:
Hello @cgillum,

Maybe I am missing something but my current understanding is that we have in general 3 options:
- `extendedSessions` is `true`, we don't log the replay events
- `extendedSessions` is `false`, here we will have the replay events available for logging but then based on the value of `logReplayEvents` we can change the behavior -> `logReplayEvents`  is `true`, we will log the replay events and `logReplayEvents` is `false` and then we will skip logging the replay even though `extendedSessions` is `false`.

So if this is true, I think we need to keep both parameters to cover all 3 cases.

Please let me what you think.

I will double check the logic.

Thank you!

Reply:
The only thing this parameter controls is whether to filter out the replay logs. The actual logging behavior is configured separately in another method. I was thinking we rename `extendedSessions` *in this context only* to `filterOutReplayLogs`. The value we use is essentially `filterOutReplayLogs = extendedSessions || !logReplayEvents`. Would that work?

Reply:
Great! Thank you very much for the clarification!